autofs-5.1.3 - fix typo in amd_parse.c

From: Ian Kent <raven@themaw.net>

Fix a typo in the "cachefs" fs type value syntax error message.

Signed-off-by: Ian Kent <raven@themaw.net>
---
 CHANGELOG           |    1 +
 modules/amd_parse.y |    2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

--- autofs-5.0.7.orig/CHANGELOG
+++ autofs-5.0.7/CHANGELOG
@@ -268,6 +268,7 @@
 - reset master map list on startup retry.
 - improve debug logging of lookup key.
 - fix cachefs parse message not being logged.
+- fix typo in amd_parse.c.
 
 25/07/2012 autofs-5.0.7
 =======================
--- autofs-5.0.7.orig/modules/amd_parse.y
+++ autofs-5.0.7/modules/amd_parse.y
@@ -299,7 +299,7 @@ option_assignment: MAP_OPTION OPTION_ASS
 			amd_msg(msg_buf);
 			YYABORT;
 		} else if (!strcmp($3, "cachefs")) {
-			sprintf(msg_buf, "file syatem %s is not "
+			sprintf(msg_buf, "file system %s is not "
 					 "supported by autofs, ignored", $3);
 			amd_msg(msg_buf);
 		} else {
